### PR TITLE
chore(messenger_communities): up unknown curated communities timeout

### PR DESCRIPTION
**Pull request by jrainville**
Needed for https://github.com/status-im/status-desktop/issues/12207

The issue is that the curation Dapp is not up yet, so we hard coded the contract with communities.
However, one of the curated communities is very old and not used anymore, so it is no longer broadcasted on Waku.

That means that we enter the "unknown community timeout" loop of 3 seconds. It ends up spamming a little bit.

I upped the timer to 1 minute for now. We can lower it back down when we deploy the real Dapp and contract.

### CODE REVIEW

File: protocol/messenger_communities.go
Comment (jrainville) [inline protocol/messenger_communities.go:507]: I can't comment below enough, but maybe we could make the `SendCuratedCommunitiesUpdate` function smarter too. Right now, it sends all the communities every time. Maybe it should only be called when there is an actual update?

**Review by osmaczko — APPROVED** [review]
Looks ok to me but please wait for @vkjr approval, maybe the lower timeout was introduced for a reason.

File: protocol/messenger_communities.go
Comment (vkjr) [reply protocol/messenger_communities.go:507]: @jrainville, thanks for the changes! Yes, you are absolutely right! There is already an issue to not send unnecessary updates #4029

**Review by vkjr — APPROVED** [review]

**Comment by vkjr** [comment]
> Looks ok to me but please wait for @vkjr approval, maybe the lower timeout was introduced for a reason.

Sorry for the late reply. 
No, the low timeout was introduced only because I believed that the community will be fetched anyway, so the quicker we check it again, the quicker mobile app will receive the information. But this is not always true as I know after this PR)

**Pull request merged**
